server/README.md

```diff
@@ -0,0 +1,49 @@
+# sam-app
+
+This template generates a sample NodeJS 12.x project configured with TypeScript using the [Serverless Application Model (SAM)](https://github.com/awslabs/serverless-application-model).
```

Comment:
The template uses `NodeJS 12`, although it build with no errors, @idomo1 will this be a problem in the future ?

Reply:
CI might break at some point, we can deal with that when/if it happens

Reply:
Not really concerned about CI/CD, I was more concerned about compatibility issues

Reply:
I don't know the particulars of the differences between Node 12 and 14/15, but I'm not that concerned we'll run into issues